pytorch/inference/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,110 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG PYTHON_VERSION=3.6.10
+ARG TS_VERSION="0.2.1=py36_0"
+ARG PT_INFERENCE_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.7.0_inference/20201030-212053/fc331964f3a79bd734e8dcf9795ed3e7952ac6b1/cpu/torch-1.7.0-cp36-cp36m-manylinux1_x86_64.whl
+ARG PT_VISION_URL=https://choidong-dev.s3-us-west-2.amazonaws.com/1.7.0/cpu/torchvision-0.8.1-cp36-cp36m-manylinux1_x86_64.whl
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
```

Comment:
Add `/usr/local/lib` to `LD_LIBRARY_PATH`, as done in https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.6.0/py3/Dockerfile.cpu#L21

Please do this on all the dockerfiles.